recipes/libnghttp3/all/conandata.yml

```diff
@@ -0,0 +1,4 @@
+sources:
+  "0.14.0":
```

Comment:
Is it possible to update to 0.15.0 ? 

https://github.com/ngtcp2/nghttp3/tags

Reply:
I'll bump the version. It was released while this was awaiting review.